Sources/XcodeGenKit/SchemeGenerator.swift

```diff
@@ -120,6 +122,8 @@ public class SchemeGenerator {
             switch target.location {
             case .project:
                 buildableName = pbxTarget.productNameWithExtension() ?? pbxTarget.name
+            case .package:
+                fatalError("Currently unsupported to use package for: \(target)")
```

Comment:
Ditto

Reply:
The solution above would resolve this as well.